examples/reading_logs_via_rule_message/reading_logs_via_rule_message.h

```diff
@@ -103,7 +103,6 @@ static void *process_request(void *data) {
     }
 
     pthread_exit(NULL);
-    return NULL;
```

Comment:
I agree with this change. The function is declared as void, so returning NULL is wring.

But maybe we should also mark the whole function as `[[noreturn]]` too.

Reply:
Thank you Mirko! I marked the functon as [[noreturn]].